### DISCUSSION TITLE
Orbot for Android v17.2.1 RC 2 (tor 0.4.8.7)

### DISCUSSION THREAD

**Discussion by n8fr8** [guardianproject/orbot-android#1087] [Announcements]
Alright folks, this is it. This is the version we are pushing out to EVERYONE. 

Thank you to all who contributed to get us here, to this big v17 transition moment, but especially:

@bitmold @tladesignz @meenbeese @syphyr @uniqx @SkewedZeppelin 
AND of course, all the amazing translators who have helped us get to more localizations than ever, include many new indigenous languages not even officially supported by Android (more on that soon)

Also, don't be surprised for now that our friend Orbi is hiding a bit... we don't want to shock people too much with our v17 update, so we will roll Orbi out slowly but surely. They are still hiding somewhere in the app though...

![Screenshot 2024-02-09 at 3 45 35 PM](https://github.com/guardianproject/orbot/assets/30851/ff8c0684-ffe1-448b-a7e7-b033a0e98222)

Here are some highlights of recent work...

9ebab6df Merge pull request #1047 from guardianproject/geoip_performance_optimization
f6df19df Merge pull request #1076 from syphyr/align_button
5ee90378 Optimization for #1046 only extract geoip and geoip6 from zip if those files dont already exist or if the app has been updated
1f168d40 Rewrite the `AppManagerActivity` file in Kotlin
11740111 Fix button on moat and custom bridges
3cd681c8 Sync minSdkVersion to match the rest of project
edabcb00 Rewrite ProxiedHurlStack file in Kotlin
177471b6 enabling snowflake proxy shouldn't start tor...
4d0b65ff Similarly re #1041, The MoreFragment wouldn't exit Orbot/OrbotVPN correctly. Fixed.
a9c6c39e Fix bug introduced from #1041 where snowflake proxy wasn't turned off This is because the KindnessFragment never specified that the Intents it sends to OrbotService weren't system intents... Fixed this here.
0206f6cb intent integrator doesnt need jar libs directory dep...
fd1f542c (dev/gomobile_ldflags) Reduce OrbotLib binary size by ~50% with go compiler ldflags -w and -s -w       disable DWARF generation -s         disable symbol table
8098d073 Bump IPtProxy to Version 3.4.0
12e3dd85 Merge pull request #1059 from guardianproject/new_langs
c373cc51 change three periods to elipsis in french + english strings
7fd38408 (tag: languagedemo, dev/new_langs) Added support for Nahuatl in language picker
4d0361e3 Added new Páez string resources
767c1550 Added new Wayuu string resources
c636fa27 Added new Guambiano string resources
918d5d6a Added Nahuatl string resources
67120132 This is actually the latest geoip used by the tor project, previously the script in orbot just pulled from tor-android...
8a9f979b Made sure incoming intent isnt null before we see if its from the system #1041 Not sure why'd you get a null intent anyway, but this way we avoid a NPE...
4d2be809 Start Orbot via always-on VPN
18481697 Ensure ICMPV6 packets are not routed
5a156bab for #1008 manually ensure ICMP packets are not routed or bypassed also tune VPN settings to ensure IPv6 traffic routing as possible

<hr /><em>This discussion was created from the release <a href='https://github.com/guardianproject/orbot/releases/tag/17.2.1-RC-1-tor-0.4.8.7'>Orbot for Android v17.2.1 RC 2 (tor 0.4.8.7)</a>.</em>

**Comment by Azad998** [comment]
Hello
You wrote in the tag section RC2
But you have put the RC1 file for download!

**Comment by ostad232** [comment]
Finally, which one isRC1 or RC2?
Please activate the new version of the webtunnel bridge.
Sometimes, even after 100%, it does not connect, nor does a site open.
Proxy only mode should be available on the first page like before. Please let the March version be a surprise.

**Comment by saskia1962** [comment]
On May 6, i got message saying Orbot not configured correctly.  So i removed it. Rebooted. Reinstalled. the reinstalled version looked like the version shown here,  however while at the DMV, i was having trouble downloading a bank statement for proof of residency so i uninstalled and reinstalled later. Now i have old version of Orbot again. i dont see a list of versions on the website so i can get the most recent.

**Comment by ShadowFox729** [comment]
Why does the link from guardian website lead here but only talks about Orbot for android but on the guardian page it mentions iOS. I haven't read anything of orbot for iOS on GitHub using that link?

**Reply by SkewedZeppelin** [reply]
the code for Orbot iOS is here: https://github.com/guardianproject/orbot-apple
